aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -260,6 +260,7 @@ Tensor& sum_out(Tensor& result, const Tensor& self, IntArrayRef dim,
   if (iter.numel() == 0) {
     result.zero_();
   } else {
+    result.zero_();
```

Comment:
Just one question, don't we want to accelerate this on device?

Reply:
Oh, it might be hb to hb copy, am I right?

Reply:
Oh okay. Calling `fill_` also does the job, I guess. Thanks for the explaining why it's needed, I couldn't totally understand it before.